internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -778,3 +778,18 @@ func convertHTTPVersion(protocolVersion *string) *string {
 	pv := strings.ToUpper(*protocolVersion)
 	return &pv
 }
+
+func convertSecrets(secrets map[string]manifest.Secret) map[string]template.Secret {
+	if len(secrets) == 0 {
+		return nil
+	}
+	m := make(map[string]template.Secret)
+	for name, mftSecret := range secrets {
+		var tplSecret template.Secret = template.SecretFromSSMOrARN(mftSecret.Value())
```

Comment:
Just curious about your choice of syntax here. Why not use the shorthand `:=` here?

Reply:
ooh because when using the shorthand it assigns the variable the concrete type `ssmOrARNSecret`, so later while trying to assign a `secretsManagerNameSecret` the compiler yells at us saying that the types mismatch.

Instead, here I'm explicitly saying use the `template.Secret` interface instead of the concrete type so that a secrets manager secret can be assigned to the same variable